ratis-docs/src/site/markdown/configurations.md

```diff
@@ -104,6 +104,16 @@ When bootstrapping a new peer, If the gap between the match index of the
 peer and the leader's latest committed index is less than this gap, we
 treat the peer as caught-up. Increase this number when write throughput is high.
 
+---------------------------------------------------------------------------------
+| **Property**    | `raft.server.bootstrap.timeout`     |
+|:----------------|:------------------------------------|
+| **Description** | timeout of bootstrapping a new peer |
+| **Type**        | TimeDuration                        |
+| **Default**     | 3 times of timeoutMax               |
```

Comment:
Thanks for updating the doc!  Let's use the full name `raft.server.rpc.timeout.max`.